calnex/cmd/reboot_test.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright (c) Facebook, Inc. and its affiliates.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"testing"
+
+	"github.com/facebook/time/calnex/api"
+	"github.com/stretchr/testify/require"
+)
+
+func TestReboot1(t *testing.T) {
```

Comment:
Normally we shouldn't need a test for the main (which in this case is cmd).
Also it's a complete duplicate of the test in `api_test`, so I don't think you need this one at all.

Reply:
So shall I remove the file `reboot_test.go` and keep the test in `api_test.go`?

Reply:
Sure will remove it